wgsl/index.bs

```diff
@@ -6934,28 +6934,30 @@ That's not a full user-defined function declaration.
   <thead>
     <tr><th>Precondition<th>Conclusion<th>Notes
   </thead>
-  <tr><td rowspan=4>
-    |e|: |T|<br>
-    |T| is [FLOATING]<br>
-    |TR| is bool if |T| is a scalar, or<br>
-    vec|N|&lt;bool&gt; if |T| is a vector
-
-    <td class="nowrap">`isNan(`|e|`) -> `|TR|
+  <tr algorithm="isNan">
+  <td rowspan=4>
+    |I| is [FLOATING]<br>
+    |T| is bool if |I| is a scalar, or<br>
+    vec|N|&lt;bool&gt; if |I| is a vector
+    <td class="nowrap">`isNan(`|e|`:` |I| `) -> ` |T|
     <td>Test for NaN according to [[!IEEE-754|IEEE-754]].<br>
-    [=Component-wise=] when |T| is a vector. (OpIsNan)
-  <tr><td class="nowrap">`isInf(`|e|`) -> `|TR|
+    [=Component-wise=] when |I| is a vector. (OpIsNan)
+  <tr algorithm="isInf">
+    <td class="nowrap">`isInf(`|e|`:` |I| `) -> ` |T|
     <td>Test for infinity according to [[!IEEE-754|IEEE-754]].<br>
-    [=Component-wise=] when |T| is a vector. (OpIsInf)
-  <tr><td class="nowrap">`isFinite(`|e|`) -> `|TR|
+    [=Component-wise=] when |I| is a vector. (OpIsInf)
+  <tr algorithm="isFinite">
+    <td class="nowrap">`isFinite(`|e|`:` |I| `) -> ` |T|
     <td>Test a finite value according to [[!IEEE-754|IEEE-754]].<br>
-    [=Component-wise=] when |T| is a vector.
-  <tr><td class="nowrap">`isNormal(`|e|`) -> `|TR|
+    [=Component-wise=] when |I| is a vector.
+  <tr algorithm="isNormal">
+    <td class="nowrap">`isNormal(`|e|`:` |I| `) -> ` |T|
     <td>Test a normal value according to [[!IEEE-754|IEEE-754]].<br>
-    [=Component-wise=] when |T| is a vector.
+    [=Component-wise=] when |I| is a vector.
 
   <tr algorithm="runtime-sized array length">
-    <td>|e|: ptr&lt;storage,array&lt;|T|&gt;&gt;
-    <td>`arrayLength(`|e|`)`: u32<td>Returns the number of elements in the [=runtime-sized=] array.<br>
+    <td>|T| is u32
+    <td>`arrayLength(`|e|`:` ptr&lt;storage,array&lt;|I|&gt;&gt; `) -> ` |T|<td>Returns the number of elements in the [=runtime-sized=] array.<br>
```

Comment:
There's no reason to parameterize the |T| here.  But |I| is left unparameterized.

Suggest: 
* no special reconditions here, because it's already constrained by what can be stored in a runtime-sized array in the 'storage' storage class
* change the conclusion to :     arrayLength(e: ptr&lt;storage,array&lt;_T_&gt;&gt;) -> u32

Reply:
Yes, you're right. I reverted this part of my change.